### DISCUSSION TITLE
Can I create an aarch64 OS/VM on an amd64 host?

### DISCUSSION THREAD

**Discussion by fadedbee** [quickemu-project/quickemu#914] [Quickemu] [answered]
I often need to quickly build aarch64 VMs on an amd64 Debian host.

Can this do that, or is it out of scope?

**Comment by flexiondotorg** [comment, marked as answer]
As of Quickemu 4.9.3, it is not possible to create and run foreign architecture VMs. This is a feature we'd like to add in the future.
